### PR TITLE
Joint Chaos and Iter8 experiment

### PR DESCRIPTION
**Pull request by sriumcp**
Doc changes that support the KubeCon 2023 talk on the joint LitmusChaos + Iter8 experiment. In particular, this provides a new tutorial for the joint experiment.

### CODE REVIEW

File: docs/tutorials/chaos/slo-validation-chaos.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+*** 
+
+## Observe experiments
+Observe the LitmusChaos and Iter8 experiments as follows. The chaos and Iter8 experiments 
+
+=== "LitmusChaos"
+    Verify that the phase of the chaos experiment is `Running`.
+    ```shell
+    kubectl get chaosresults/litmuschaos-httpbin-pod-delete -n default \
+    -ojsonpath='{.status.experimentStatus.phase}'
```
Comment (Alan-Cha) [inline docs/tutorials/chaos/slo-validation-chaos.md:100]: Is there supposed to be a space?

```suggestion
    -o jsonpath='{.status.experimentStatus.phase}'
```

File: docs/tutorials/chaos/slo-validation-chaos.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    ```
+
+    ??? note "On completion of the LitmusChaos experiment"
+        After the LitmusChaos experiment completes (in ~3600 sec), the phase of the experiment will change to `Completed`. At that point, you can verify that the chaos experiment returns a `Pass` verdict. The `Pass` verdict states that the application is still running after chaos has ended.
+
+        When the LitmusChaos experiment is still running, its verdict will be set to `Awaited`.
+
+        ```shell
+        kubectl get chaosresults/litmuschaos-httpbin-pod-delete -n default \
+        -o=jsonpath='{.status.experimentStatus.verdict}'
```
Comment (Alan-Cha) [inline docs/tutorials/chaos/slo-validation-chaos.md:110]: See above? Inconsistent.

```suggestion
        -o=jsonpath='{.status.experimentStatus.verdict}'
```

**Review by Alan-Cha — CHANGES_REQUESTED** [review]

File: docs/tutorials/chaos/slo-validation-chaos.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    --repo https://iter8-tools.github.io/hub/ \
+    --set applabel='app=httpbin' \
+    --set totalChaosDuration=3600 \
+    --set chaosInterval=5
+    ```
+
+    ??? note "About this LitmusChaos experiment"
+        This is a [LitmusChaos pod-delete experiment](https://litmuschaos.github.io/litmus/experiments/categories/pods/pod-delete/) packaged for reusability in the form of a Helm chart. This experiment causes (forced/graceful) pod failure of specific/random replicas of application resources, in this case, pods with a label called `app` with value `httpbin`.
+
+        The deletion of pod(s) will be attempted by the chaos experiment once every  `chaosInterval`(5) seconds, and this experiment will terminate after `totalChaosDuration`(3600) seconds.
```
Comment (kalantar) [inline docs/tutorials/chaos/slo-validation-chaos.md:69]: add space before `(5)` and `(3600)`

File: docs/tutorials/chaos/slo-validation-chaos.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        spec:
+          containers:
+          - name: httpbin
+            image: kennethreitz/httpbin
+            ports:
+            - containerPort: 80
+          initContainers:
+          - name: init-myservice
+            image: busybox:1.28
+            command: ['sh', '-c', 'sleep 1']
```
Comment (kalantar) [inline docs/tutorials/chaos/slo-validation-chaos.md:41]: This container seems to be the difference between this and the way we start _httpbin_ in the other experiments (`kubectl create deploy httpbin --image=kennethreitz/httpbin --port=80`). Is this addition necessary? Does it need explanation especially after being told to try _your first experiment_

Comment (sriumcp) [reply docs/tutorials/chaos/slo-validation-chaos.md:41]: The deployment is structured purely for moving the demo narrative forward ... 

We want SLOs to fail when scale = 1, and we want SLOs to succeed when scale = 3.

Structuring the deployment as shown in this example seems to achieve this effect predictably well.

***

I don't think we need to explain the above for the end-user, since that is irrelevant to the purpose of the tutorial (which is here's how you combine chaos and Iter8 experiments to test your app).

Comment (sriumcp) [reply docs/tutorials/chaos/slo-validation-chaos.md:41]: More important, the user is expected to replace the `httpbin` deployment in this tutorial, with their own app (and I believe, the explanations under expandable sections provide enough details for the user to accomplish this).

File: docs/tutorials/chaos/slo-validation-chaos.md (hunk 1 above)
Comment (sriumcp) [reply docs/tutorials/chaos/slo-validation-chaos.md:100]: I verified that the tutorial works as written.

File: docs/tutorials/chaos/slo-validation-chaos.md (hunk 2 above)
Comment (sriumcp) [reply docs/tutorials/chaos/slo-validation-chaos.md]: Fixed.

File: docs/tutorials/chaos/slo-validation-chaos.md (hunk 3 above)
Comment (sriumcp) [reply docs/tutorials/chaos/slo-validation-chaos.md]: Fixed.

File: docs/user-guide/tasks/ready.md [hunk 5]
```diff
@@ -25,6 +25,7 @@ iter8 k launch \
 | deploy  | string | Name of a Kubernetes deployment. The task checks if the deployment exists and its `Available` condition is set to true. |
 | service | string | Name of a Kubernetes service. The task checks if the service exists. |
 | ksvc | string | Name of a Knative service. The task checks if the service exists and its `Ready` condition is set to true. |
+| chaosengine | string | Name of a LitmusChaos chaosengine resource object. The task checks if the object exists. |
```
Comment (Alan-Cha) [inline docs/user-guide/tasks/ready.md:28]:
```suggestion
| chaosengine | string | Name of a LitmusChaos ChaosEngine resource object. The task checks if the object exists. |
```

File: docs/tutorials/chaos/slo-validation-chaos.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+=== "LitmusChaos"
+    Uninstall LitmusChaos from your cluster as described [here](https://docs.litmuschaos.io/docs/user-guides/uninstall-litmus/).
+
+***
+
+??? note "Some variations and extensions of this experiment"
+    1. Reuse the above experiment with *your* app by replacing the `httpbin` app with *your* app, and modifying the experiment values appropriately.
+    2. Iter8 supports load testing and SLO validation for [gRPC services](../load-test-grpc.md). Try a joint chaos injection and SLO validation experiment for gRPC.
+    3. Litmus makes it possible to inject [over 51 types of Chaos](https://hub.litmuschaos.io/). Modify the LitmusChaos Helm chart with new templates in order to use any of these other types of chaos experiments.
```
Comment (Alan-Cha) [inline docs/tutorials/chaos/slo-validation-chaos.md:177]:
```suggestion
    3. Litmus makes it possible to inject [over 51 types of chaos](https://hub.litmuschaos.io/). Modify the LitmusChaos Helm chart with new templates in order to use any of these other types of chaos experiments.
```

File: docs/tutorials/chaos/slo-validation-chaos.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+***
+
+## Scale app and retry
+Scale up the app so that replica count is increased to 3. 
+```shell
+kubectl scale --replicas=3 -n default deploy/httpbin
+```
+
+The scaled app is now more resilient. Performing the same experiments as above will now result in SLOs being satisfied and a winner being found. [Relaunch the experiments](#launch-experiments) and [observe the experiments](#observe-experiments). You should now find that SLOs are satisfied.
```
Comment (Alan-Cha) [inline docs/tutorials/chaos/slo-validation-chaos.md:150]:
```suggestion
The scaled app is now more resilient. Performing the same experiments as above will now result in SLOs being satisfied and a winner being found. [Relaunch the experiments](#launch-experiments) and [observe the experiments](#observe-experiments). You should now find that the SLOs are satisfied.
```

**Review by Alan-Cha — APPROVED** [review]
